examples/web.rs

```diff
@@ -1,3 +1,4 @@
+#![deny(warnings)]
```

Comment:
Please remove these from all of the examples. I'd rather folks be able to compile examples even if there are warnings.